libs/core/src/lib/tabs/tab-list.component.ts

```diff
@@ -133,7 +151,7 @@ export class TabListComponent implements AfterContentInit, OnChanges, OnDestroy
         return result;
     }
 
-    private resetTabHook(): void {
+    private _resetTabHook(): void {
         this.selectedIndex = 0;
```

Comment:
This `this.selectedIndex = 0;` is doubled in `selectTab` method.

Reply:
Done